src/System.Runtime.Analyzers/Core/InitializeStaticFieldsInline.Fixer.cs

```diff
@@ -27,10 +29,10 @@ public sealed override FixAllProvider GetFixAllProvider()
         }
 
         public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
-        {                              
+        {
+            // TODO: Implement the fixer.
             // This is to get rid of warning CS1998, please remove when implementing this analyzer
             await new Task(() => { });
```

Comment:
The fix would be a bit involved because it would need to find all static fields being assigned in the constructor and make them all as initializers. It is a mechanical transformation and saves a bunch of copy\pasting - sounds useful actually.